<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintUIClass</class>
 <widget class="QMainWindow" name="PaintUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1569</width>
    <height>1168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_pbsys">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_open">
           <property name="text">
            <string>打开图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_save">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_magnify">
           <property name="text">
            <string>放大</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_shrink">
           <property name="text">
            <string>缩小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_reposition">
           <property name="text">
            <string>复位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_redraw">
           <property name="text">
            <string>重画</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_clear">
           <property name="text">
            <string>清屏</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>170</width>
         <height>1087</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1569</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
